R/data.R

```diff
@@ -31,6 +31,9 @@
 ##'
 ##' @param fit_to_variants Logical, whether to fit to variants data or not
 ##'
+##' @param fit_to_under25 Logical, whether to include data on under 25s when
```

Comment:
`fit_to_under25` is only being used as `TRUE` in the fits - do we need to keep it?

Reply:
Thanks, done